app/src/main/java/com/example/minerva/data/repository/Repository.kt

```diff
@@ -0,0 +1,62 @@
+package com.example.minerva.data.repository
+
+import com.example.minerva.data.local.FavouriteArticleDao
+import com.example.minerva.data.local.NewsDao
+import com.example.minerva.data.local.UserDao
+import com.example.minerva.data.model.FavouriteArticle
+import com.example.minerva.data.model.NewsDto
+import com.example.minerva.data.model.User
+import com.example.minerva.data.remote.RetrofitService
+import dagger.Module
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import kotlinx.coroutines.flow.Flow
+import retrofit2.Response
+
+
+@Module
+@InstallIn(SingletonComponent::class)
+class Repository(
```

Comment:
here we don't need the @Module! 
you can just say:
`@Singleton
class Repository @Inject constructor(//constructor params)`